bridge/opentracing/internal/mock.go

```diff
@@ -85,7 +85,14 @@ func (t *MockTracer) Start(ctx context.Context, name string, opts ...oteltrace.S
 		startTime = time.Now()
 	}
 	spanKind := spanOpts.SpanKind
-	if spanKind == "" {
+	switch spanKind {
```

Comment:
Change this to use the `apitrace.ValidateSpanKind` helper.

Reply:
Of course, my mistake. :)